.goreleaser.yml

```diff
@@ -137,14 +62,8 @@ archives:
   - id: sat
     name_template: 'sat-v{{ .Version }}-{{ .Os }}-{{ .Arch }}'
     builds:
-      - sat-linux-amd64
-      - sat-linux-arm64
-      - sat-darwin-amd64
-      - sat-darwin-arm64
+      - sat
   - id: constd
     name_template: 'constd-v{{ .Version }}-{{ .Os }}-{{ .Arch }}'
     builds:
-      - constd-linux-amd64
-      - constd-linux-arm64
-      - constd-darwin-amd64
-      - constd-darwin-arm64
+      - constd
```

Comment:
This is referencing the modified one on line 50 with the overridden values, right?

Reply:
Excatly!